openjdk/build.gradle

```diff
@@ -153,19 +152,37 @@ model {
 
     tasks {
         $.binaries.withType(SharedLibraryBinarySpec).each {
+            // Build the native artifact classifier from the OS and architecture.
+            def osName
+            def os = it.targetPlatform.operatingSystem
+            if (os.isWindows()) {
+                osName = "windows"
+            } else if (os.isMacOsX()) {
+                osName = "osx"
+            } else if (os.isLinux()) {
+                osName = "linux"
+            } else if (os.isFreeBSD()) {
+                osName = "freebsd"
+            } else if (os.isSolaris()) {
+                osName = "solaris"
```

Comment:
I don't think BoringSSL supports FreeBSD or Solaris, so I don't want to confuse the issue by adding them here.